### ISSUE TITLE
Issue evaluating

### ISSUE THREAD

**Issue by jake-shomer** [potterhsu/SVHNClassifier-PyTorch#4]
The model's evaluation accuracy maintains 0.00% throughout the entire training process. The model's loss decreases significantly but when evaluating on the validation set it returns 0% accuracy. No code was modified to perform training. Any idea of what the issue may be?

**Comment by muety** [comment]
I got the same problem.
Any solutions?

**Comment by potterhsu** [comment]
If you’re using Python 2, try to replace with `float(len(self._loader.dataset))` at `evaluator.py` #47, or you could check the value of `num_corrects`
